providers/tailscale/resources/tailscale.lr

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+option provider = "go.mondoo.com/cnquery/v11/providers/tailscale"
+option go_package = "go.mondoo.com/cnquery/v11/providers/tailscale/resources"
+
+// The Tailscale provider
+tailscale {
+  // List devices in a tailnet
+  devices() []tailscale.device
+}
+
+// A Tailscale device (sometimes referred to as node or machine)
+tailscale.device @defaults("id hostname os") {
+  init(id? string)
+  // The legacy identifier for a device
+  id string
+  // The preferred identifier for a device (not supported yet)
+  // nodeId string
+  // The machine name in the admin console
+  hostname string
+  // The operating system that the device is running
+  os string
+  // The MagicDNS name of the device
+  name string
+  // The user who registered the node. For untagged nodes, this user is the device owner
+  user string
+  // Lets you assign an identity to a device that is separate from human users, and use it as part of an ACL to restrict access
+  tags []string
+  // A list of Tailscale IP addresses for the device, including both IPv4 and IPv6 addresses
+  addresses []string
+  // The version of the Tailscale client software; this is empty for external devices
+  clientVersion string
+  // For internal use and is not required for any API operations. This value is empty for external devices
+  machineKey string
+  // Mostly for internal use, required for select operations, such as adding a node to a locked tailnet
+  nodeKey string
+  // Indicates an issue with the tailnet lock node-key signature on this device. This field is only populated when tailnet lock is enabled
+  tailnetLockError string
+  // The node's tailnet lock key
+  tailnetLockKey string
+  // 'true' if the device is not allowed to accept any connections over Tailscale, including pings
+  blocksIncomingConnections bool
+  // 'true' if the device has been authorized to join the tailnet; otherwise, 'false'
+  authorized bool
+  // 'true' indicates that a device is not a member of the tailnet, but is shared in to the tailnet; if 'false', the device is a member of the tailnet
+  isExternal bool
+  // 'true' if the keys for the device will not expire
+  keyExpiryDisabled bool
+  // 'true' if a Tailscale client version upgrade is available. This value is empty for external devices
+  updateAvailable bool
+  // The date on which the device was added to the tailnet; this is empty for external devices
+  created time
```

Comment:
@tas50 I think we want to harmonize on `createdAt`, correct?

Reply:
That seems to be the one we have the most of so yeah let's do that

Reply:
Updating all time fields. 💯